### ISSUE TITLE
Globally added CSS-rule i.icon messes with Search-Icon

### ISSUE THREAD

**Issue by sebastian-marinescu** [modmore/QuickstartButtons#35]
A globally added rule for i.icon adds a width of 16px.

```
i.icon {
    display: inline-block;
    width: 16px;
    height: 16px;
    background-size: 100%;
    background-repeat: no-repeat;
    background-position: left center;
}
```

It seems to be used for the configuration modal, 
but it messes up the search-icon when on the dashbard: 

![image](https://cloud.githubusercontent.com/assets/3798871/13216802/78b5a576-d95e-11e5-9755-4cb5aa1b1651.png)

**Comment by sebastian-marinescu** [comment]
It also messes a little bit with the other icons in the header: 

![image](https://cloud.githubusercontent.com/assets/3798871/13334723/c86c8e84-dc0d-11e5-866d-3a5505681988.png)
_becomes_
![image](https://cloud.githubusercontent.com/assets/3798871/13334683/98166598-dc0d-11e5-911f-213297393ea9.png)

And the height messes a little with the tree-icons:

![image](https://cloud.githubusercontent.com/assets/3798871/13334778/f61f32dc-dc0d-11e5-983d-24629632175f.png)  >  ![image](https://cloud.githubusercontent.com/assets/3798871/13334784/f9b919bc-dc0d-11e5-8605-35a86943708f.png)

Although I like your line-height in the tree better, I don't think it should come from QSB. 
I have checked the icons of QSB in the dashboard-widget and in the CMP, both use `.fa-icon`.

**Comment by Mark-H** [comment]
Fixed in 1.3
